build.gradle.kts

```diff
@@ -25,6 +32,8 @@ dependencies {
   // Integration test dependencies
   integrationTestImplementation("com.h2database:h2")
 
+  implementation("org.springdoc:springdoc-openapi-ui:1.7.0")
```

Comment:
Minor. and I accept this is a subjective style thing .... but is it worth grouping all the dependency groups together? IE. put all the main implementations together, all the runtime deps together, all the implement test deps together, all the test deps together etc?
If so, would this line be better off somewhere near line 26 ?